src/stats.c

```diff
@@ -433,6 +434,16 @@ stats_arena_print(void (*write_cb)(void *, const char *), void *cbopaque,
 		    "resident:                %12zu\n", resident);
 	}
 
+	CTL_M2_GET("stats.arenas.0.tcache_bytes", i, &tcache_bytes, size_t);
+	if (json) {
+		malloc_cprintf(write_cb, cbopaque,
+		    "\t\t\t\t\"tcache\": %zu,\n", tcache_bytes);
```

Comment:
Hmm. If we put this segment behind "if (config_tcache)", the ending comma print logic becomes quite tricky (as it's possible for resident above to be the last one). What do you think about putting tcache_bytes above resident so that its not the last one?